### PR TITLE
Update devcontainer image

### PR DESCRIPTION
**Pull request by bells17**
## What does this PR do?

Update the devcontainer image to the following image: `mcr.microsoft.com/devcontainers/go:1.22-bookworm`.

## Why is it important?

Currently, we cannot open the project using devcontainer because the devcontainer image `mcr.microsoft.com/devcontainers/go:0-1.21-bullseye` cannot be found.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // README at: https://github.com/devcontainers/templates/tree/main/src/go
 {
 	"name": "Go",
-	"image": "mcr.microsoft.com/devcontainers/go:0-1.21-bullseye",
+	"image": "mcr.microsoft.com/devcontainers/go:1.22-bookworm",
```
Comment (mdelapenya) [inline .devcontainer/devcontainer.json:5]: Could you use a Go 1.21 image? We set the base version as the lowest supported Go version, and we bump it whenever there is an update in the Go EOL (https://endoflife.date/go)

**Review by mdelapenya — COMMENTED** [review]
Thanks for submitting the changes! Could you take a look at the Go version 🙏 ?

Cheers

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (bells17) [reply .devcontainer/devcontainer.json]: @mdelapenya Thank you for reviewing my changes.
I've changed the Go version, so could you please take another look?

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks!

**Comment by mdelapenya** [comment]
The influx error is not related: it will be fixed with #2532

**Pull request merged**